sky/packages/sky/lib/widgets/scrollable.dart

```diff
@@ -459,7 +460,12 @@ abstract class ScrollableWidgetList extends Scrollable {
         } else {
           drawStart = paddedScrollOffset;
         }
-        itemShowIndex = math.max(0, (drawStart / itemExtent).floor());
+        if (itemCount > 0) {
+          // floor(epsilon) = 0, floor(-epsilon) = -1, so:
+          if (drawStart.abs() < _kEpsilon)
+            drawStart = 0.0;
```

Comment:
Should we change the animations toward zero to explicitly hit zero when they complete?  We do something similar for other animations where we explicitly make sure they hit their `end` value exactly.